modMcf/src/org/aion/mcf/config/CfgLog.java

```diff
@@ -104,6 +104,8 @@ public String toXML() {
              * Boolean value to allow logger to be toggled ON and OFF
              */
             xmlWriter.writeCharacters("\t\t");
+            xmlWriter.writeCharacters("<!--Enable/Disable logback service; if disabled, output will not be logged -->");
```

Comment:
you need to use `writeComment` here like in https://github.com/aionnetwork/aion/blob/1bbeec193eda2a915233faa51aa69680d5bcfd43/modMcf/src/org/aion/mcf/config/CfgDb.java#L195-L200